### PR TITLE
flashrom: Try harder to convert to a correct verfmt semver

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: libfwupdplugin/fu-common-version.c [hunk 1]
```diff
@@ -235,6 +235,73 @@ _g_ascii_is_digits(const gchar *str)
 	return TRUE;
 }
 
+static guint
+_fwupd_version_format_sections(FwupdVersionFormat fmt)
```
Comment (superm1) [inline libfwupdplugin/fu-common-version.c:239]: nit; it's not really format sections - it's count the "number" of sections

**Review by superm1 — DISMISSED** [review]
one nit and the commit message should be fixed up - this isn't flashrom (but it might be _for_ that)
otherwise LGTM

File: libfwupdplugin/fu-common-version.c (hunk 1 above)
Comment (hughsie) [reply libfwupdplugin/fu-common-version.c:239]: _fwupd_version_format_number_sections perhaps?

Comment (superm1) [reply libfwupdplugin/fu-common-version.c:239]: thats good

**Pull request merged**